masu/external/accounts_accessor.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Provider external interface for koku to consume."""
+
+
+from masu.external.db.cur_accounts_db import CURAccountsDB
+
+
+class AccountsAccessorError(Exception):
+    """Cost Usage Report Accounts error."""
+
+    pass
+
+
+# pylint: disable=too-few-public-methods
+class AccountsAccessor:
+    """Interface for masu to use to get CUR accounts."""
+
+    def __init__(self, source='db'):
```

Comment:
maybe update the param to `source_type`; seem strange to assign source to `self.source_type` and then assign `self. source` to something else.